Microsoft.Azure.Devices.Edge.sln

```diff
@@ -191,6 +191,12 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.Devices.Edg
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.Azure.Devices.Edge.Agent.Kubernetes.IntegrationTest", "edge-agent\test\Microsoft.Azure.Devices.Edge.Agent.Kubernetes.IntegrationTest\Microsoft.Azure.Devices.Edge.Agent.Kubernetes.IntegrationTest.csproj", "{79540E35-45EA-4568-9AB7-C1CC0BFF73EE}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "edge-modules", "edge-modules", "{69157D2B-2AD3-4EE0-A837-6AD9329EDDC3}"
```

Comment:
why do we need to add this?

Reply:
I'm pretty sure this was added when I used the dotnet command to add TwinTester to the solution. I could remove it, but I'm hesitant to touch generated code (unless you think I added something else by mistake).